common/views/components/ImageViewer/ImageViewer2.js

```diff
@@ -188,18 +190,14 @@ class ImageViewer extends React.Component<ImageViewerProps, ImageViewerState> {
             }
           }
         </Transition>
-        <Transition in={this.state.showViewer} timeout={{enter: 0, exit: 700}}>
-          {
-            (status) => {
-              return <ViewerContent
-                classes={`scale scale-${status}`}
-                viewerVisible={this.state.showViewer}
-                id={this.props.id}
-                contentUrl={this.props.contentUrl}
-                handleViewerDisplay={this.handleViewerDisplay} />;
-            }
-          }
-        </Transition>
+        {this.state.showViewer &&
+          <ViewerContent
+            classes=''
```

Comment:
Is this more/less preferable to making `classes` an optional prop?

Reply:
I might just remove this, it was there for the scale classes.